cloud/cluster/cluster.go

```diff
@@ -2,30 +2,31 @@ package cluster
 
 import (
 	"sort"
-	"time"
 )
 
 // Cluster represents a cluster of Nodes.
 type Cluster struct {
 	state    *state
 	reqCh    chan interface{}
-	ch       chan interface{}
-	updateCh chan []NodeUpdate
+	updateCh chan ClusterUpdate
 	subs     []chan []NodeUpdate
 }
 
+// Clusters can be updated in two ways:
+// *) a new state of the Cluster, which is a []Node
+// *) updates to specific Nodes, which is a []NodeUpdate
+type ClusterUpdate interface{}
```

Comment:
chan Nodes is not the same type as chan cluster.ClusterUpdate

So just making the Cron wouldn't be possible because it returns a chan cluster.ClusterUpdate